templates/web/base/admin/reports/_edit_waste.html

```diff
@@ -161,6 +161,7 @@ <h2>Payment information</h2>
     [ 'Pro rata payment', 'pro_rata' ],
     [ 'Admin fee', 'admin_fee' ],
     [ 'Payment method', 'payment_method' ],
+    [ 'Cheque reference', 'chequeReference', 1 ],
     [ 'DD reference', 'payerReference', 1 ],
     [ 'SCP reference', 'scpReference', 1 ],
     [ 'Last payment method', 'LastPayMethod' ],
```

Comment:
Unrelated to this but I notice that this just lists the number in the admin which I'm not sure is entirely helpful?